pydis_site/apps/api/tests/test_models.py

```diff
@@ -34,6 +35,43 @@ def test_shows_attributes(self):
         self.assertEqual(repr(self.klass), expected)
 
 
+class NitroMessageLengthTest(TestCase):
+    def setUp(self):
+        self.user = User.objects.create(id=50, name='bill', discriminator=5)
+        self.context = MessageDeletionContext.objects.create(
+            id=50,
+            actor=self.user,
+            creation=dt.utcnow()
+        )
+
+    def test_create(self):
+        message = DeletedMessage(
+            id=46,
+            author=self.user,
+            channel_id=666,
+            content="w"*4000,
+            deletion_context=self.context,
+            embeds=[]
+        )
+
+        try:
+            message.clean_fields()
+        except Exception as e:
+            self.fail(f"Creation of message of length 3950 failed with: {e}")
```

Comment:
This except block is causing the 100% coverage to fail because it is never run. Can you please obliterate it?

Reply:
See #539

Reply:
As discussed in dev-core, we should add a pragma no cover here, so this specific branch is ignored, but we keep coverage on other tests